### PR TITLE
Add documentation for downloading the latest builds

### PR DESCRIPTION
**Pull request by alex-taffe**
This change is [Reviewable](https://reviewable.io/reviews/compnerd/swift-build/301)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,3 +141,16 @@ targets, or build on Windows natively without cross-compilation.
       - SDK
         - Linux
 </details>
+
+## Getting the latest build
+### Stable builds
+The latest stable build can be acuqired from the [releases](https://github.com/compnerd/swift-build/releases) page.
+### Dev builds
+The `utilities/swift-build.py` script allows dopwnloading of the latest build artifacts. The script requires the `azure-devops` and `tabulate` python packages. This can be installed with pip:
```
Comment (MaxDesiatov) [inline README.md:149]:
```suggestion
The `utilities/swift-build.py` script allows downloading of the latest build artifacts. The script requires the `azure-devops` and `tabulate` python packages. These can be installed with `pip`:
```

Comment (alex-taffe) [reply README.md:149]: Oops thanks for the typo correction

**Comment by compnerd** [comment]
Thanks @alex-taffe!  This is definitely a good idea.  For some reason the rich review indicates that the markdown isn't rendering properly.  Is that just an artifact of the diff viewer?

**Comment by ataffeqeexo** [comment]
It seems like it's just the diff viewer, it was screwed up in the first commit but I fixed it in d8899b5

**Comment by compnerd** [comment]
Sounds good to me!  I'm not going to bother with the CI for the docs change.

**Pull request merged**